Cargo.lock

```diff
@@ -3807,9 +4343,9 @@ dependencies = [
 
 [[package]]
 name = "elliptic-curve"
-version = "0.13.6"
+version = "0.13.8"
```

Comment:
Are these version bumps expected? Would you mind regenerating the `Cargo.lock` file based on the latest revision on the main branch to keep the changes minimal?

Reply:
Please do this
```
# bring in the current latest file
git checkout main -- Cargo.lock
# run any cargo command to update Cargo.lock
cargo check
```